scripts/conversion_script.py

```diff
@@ -19,15 +19,28 @@
 # Revert the `STATUS_CODE` dictionary to map number: name instead of name:
 # number as used originally.
 STATUS_CODE_NAME = {number: name for name, number in STATUS_CODE.items()}
-_C2R_LOG_FOLDER = "/var/log/convert2rhel"
+# Log folder path for convert2rhel
+C2R_LOG_FOLDER = "/var/log/convert2rhel"
+# Log file for convert2rhel
+C2R_LOG_FILE = "%s/convert2rhel.log" % C2R_LOG_FOLDER
 # Path to the convert2rhel report json file.
-C2R_REPORT_FILE = "%s/convert2rhel-pre-conversion.json" % _C2R_LOG_FOLDER
+C2R_REPORT_FILE = "%s/convert2rhel-pre-conversion.json" % C2R_LOG_FOLDER
 # Path to the convert2rhel report textual file.
-C2R_REPORT_TXT_FILE = "%s/convert2rhel-pre-conversion.txt" % _C2R_LOG_FOLDER
+C2R_REPORT_TXT_FILE = "%s/convert2rhel-pre-conversion.txt" % C2R_LOG_FOLDER
 # Path to the archive folder for convert2rhel.
-C2R_ARCHIVE_DIR = "%s/archive" % _C2R_LOG_FOLDER
+C2R_ARCHIVE_DIR = "%s/archive" % C2R_LOG_FOLDER
+# Set of yum transactions that will be rolled back after the operation is done.
 YUM_TRANSACTIONS_TO_UNDO = set()
 
+# Define regex to look for specific errors in the rollback phase in
+# convert2rhel.
+DETECT_ERROR_IN_ROLLBACK_PATTERN = re.compile(
+    r".*(error|failed|fail|failure|denied|traceback|couldn't find a backup)",
```

Comment:
```suggestion
    r".*(error|fail|denied|traceback|couldn't find a backup)",
```
The `fail` should find also occurences of `failed` and `failure`.